src/async/pingfilter.rs

```diff
@@ -0,0 +1,109 @@
+use super::imports::*;
+use websocket::{OwnedMessage, WebSocketError};
+
```

Comment:
do we need this? its nice but i don't know if we ever had to handle this in the sync version